backend/tests/core/test_exceptions.py

```
@@ -147,6 +147,25 @@ class TestModel(BaseModel):
+                    "type": "model_attributes_type",
+                    "loc": ("body",),
+                    "msg": "Input should be a valid dictionary",
+                    "input": b'{"task_id": 123}',
+                }
+            ]
+        )
+
+        response = await validation_exception_handler(request=None, exc=exc)
+
+        assert response.status_code == status.HTTP_422_UNPROCESSABLE_ENTITY
+        response_body = eval(response.body.decode())
```

Ruff (0.15.13)

[error] Use of possibly insecure function; consider using `ast.literal_eval`

(S307)